src/realm/sync/noinst/changeset_index.hpp

```diff
@@ -302,8 +302,7 @@ inline bool is_container_instruction(const sync::Instruction& instr) noexcept
 
     return mpark::holds_alternative<Instruction::ArrayInsert>(v) ||
            mpark::holds_alternative<Instruction::ArrayMove>(v) ||
-           mpark::holds_alternative<Instruction::ArrayErase>(v) ||
-           mpark::holds_alternative<Instruction::ArrayClear>(v);
+           mpark::holds_alternative<Instruction::ArrayErase>(v) || mpark::holds_alternative<Instruction::Clear>(v);
```

Comment:
[nit] why not put the `Clear` on its own line like before so it's aligned with the rest?